PMSCore/Tester.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace PMSCore
+
+{
+    /// <summary>
+    /// A Tester user who managages test cases and bug reporting.
+    /// </summary>
+    public class Tester : UserBase
+    {
+        
+
+        // Properties specific to a Tester
+        public List<string> TestCases;
+        public List<(string Description, string Severity)> BugReports;
```

Comment:
These two lines (12-13): public List<string> TestCases;
                                       public List<(string Description, string Severity)> BugReports;
                                       
TestCases and BugReports might need to be changed from public fields to properties with private setters to achieve encapsulation.